Upgrade bootstrap to 3.4.0 in `wicket-8.x` branch

As discussed in #773

I think the upgrade of the Modernizr lib from 2.8.3 to 2.8.3-1 introduced a bug:
* the webjars artifact for modernizr-2.8.3-1 only contains modernizr.min.js, but no modernizr.js
* in de.agilecoders.wicket.core.markup.html.references.ModernizrJavaScriptReference the name is set to "/modernizr/current/modernizr.js"
* in our application this leads to a MissingResourceException, because the modernizr resource can't be found:

Caused by: java.util.MissingResourceException: Unable to get cache key for [webjars js resource] /modernizr/current/modernizr.js (resolved name: /webjars/modernizr/null/modernizr.js)
	at org.apache.wicket.resource.bundles.ConcatBundleResource.reportError(ConcatBundleResource.java:263)
	at org.apache.wicket.resource.bundles.ConcatBundleResource.getCacheKey(ConcatBundleResource.java:238)
	at org.apache.wicket.request.resource.caching.version.CachingResourceVersion.getVersion(CachingResourceVersion.java:101)
	at org.apache.wicket.request.resource.caching.FilenameWithVersionResourceCachingStrategy.decorateUrl(FilenameWithVersionResourceCachingStrategy.java:105)
	at org.apache.wicket.core.request.mapper.BasicResourceReferenceMapper.mapHandler(BasicResourceReferenceMapper.java:234)
	at org.apache.wicket.request.mapper.ParentPathReferenceRewriter.mapHandler(ParentPathReferenceRewriter.java:92)
	at org.apache.wicket.request.mapper.CompoundRequestMapper.mapHandler(CompoundRequestMapper.java:210)
	at org.apache.wicket.request.cycle.RequestCycle.mapUrlFor(RequestCycle.java:451)
	at org.apache.wicket.request.cycle.RequestCycle.urlFor(RequestCycle.java:551)
	... 84 common frames omitted

--- a/bootstrap-core/src/main/java/de/agilecoders/wicket/core/settings/IBootstrapSettings.java
+++ b/bootstrap-core/src/main/java/de/agilecoders/wicket/core/settings/IBootstrapSettings.java
@@ -19,7 +19,7 @@ public interface IBootstrapSettings {
     /**
      * The version of modernizr
      */
-    String MODERNIZR_VERSION = "2.8.3-1";
+    String MODERNIZR_VERSION = "2.8.3";
 
     /**
      * The version of Popper.js

